src/constants/response.constant.ts

```diff
@@ -0,0 +1,6 @@
+export const RESPONSE = {
+  FACILITY_TERM_CREATED: 'Facility Term created',
+  FACILITY_TERM_NOT_CREATED: 'Facility Term not created',
```

Comment:
I don't think this value is used

Reply:
good spot - I was using it previously, but reverted the functionality and forgot to take this out.

Reply:
Removed old constant